hs_core/views/utils.py

```diff
@@ -21,9 +23,12 @@ def upload_from_irods(username, password, host, port, zone, irods_fname, res_fil
     try:
         irods_storage = IrodsStorage()
         irods_storage.set_user_session(username=username, password=password, host=host, port=port, zone=zone)
+        size = irods_storage.size(irods_fname)
+        if size > file_size_limit:
```

Comment:
This seems to duplicate the file size validation logic found in hs_core.hydroshare.resource.check_resource_files().  This validation should be defined in a single location in the code.